GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -184,9 +184,9 @@ private void InitializeComponent()
             this.toolbarCommit.SuspendLayout();
             this.commitStatusStrip.SuspendLayout();
             this.SuspendLayout();
-            //
+            // 
```

Comment:
Have you added the trailing spaces on purpose?

Reply:
No, this is the "designer special" 😒 
/cc: @merriemcgaw